workflow-examples/src/test/java/com/redhat/parodos/examples/integration/KubeapiWorkFlow.java

```diff
@@ -0,0 +1,130 @@
+					.getWorkFlowDefinitionById(simpleSequentialWorkFlowDefinitions.get(0).getId().toString());
+
+			// EXECUTE WORKFLOW
+			WorkflowApi workflowApi = new WorkflowApi();
+
+			// 1 - Define WorkRequests
+			WorkRequestDTO workGet = WorkRequestDTO.builder().workName(taskName)
+					.arguments(Arrays.asList(new ArgumentRequestDTO().key("api-group").value(""),
+							new ArgumentRequestDTO().key("api-version").value("v1"),
+							new ArgumentRequestDTO().key("kind-plural-name").value("configmaps"),
+							new ArgumentRequestDTO().key("operation").value("get"),
+							new ArgumentRequestDTO().key("resource-name").value("my-cm"),
+							new ArgumentRequestDTO().key("resource-namespace").value("test"),
+							new ArgumentRequestDTO().key("work-ctx-key").value("theKey")))
+					.build();
+
+			WorkRequestDTO workCreate = WorkRequestDTO.builder().workName(taskName).arguments(Arrays.asList(
+					new ArgumentRequestDTO().key("api-group").value(""),
+					new ArgumentRequestDTO().key("api-version").value("v1"),
+					new ArgumentRequestDTO().key("kind-plural-name").value("configmaps"),
+					new ArgumentRequestDTO().key("operation").value("create"),
+					new ArgumentRequestDTO().key("resource-json").value("{"
+							+ "\"apiVersion\": \"v1\",\"data\": {\"a\": \"vala\",\"b\": \"valb\"},\"kind\": \"ConfigMap\","
+							+ "\"metadata\": {\"name\": \"my-cm-create\",\"namespace\": \"test\"}}")))
+					.build();
+
+			WorkRequestDTO workUpdate = WorkRequestDTO.builder().workName(taskName).arguments(Arrays.asList(
+					new ArgumentRequestDTO().key("api-group").value(""),
+					new ArgumentRequestDTO().key("api-version").value("v1"),
+					new ArgumentRequestDTO().key("kind-plural-name").value("configmaps"),
+					new ArgumentRequestDTO().key("operation").value("update"),
+					new ArgumentRequestDTO().key("resource-json").value("{"
+							+ "\"apiVersion\": \"v1\",\"data\": {\"a\": \"valb\",\"b\": \"vala\"},\"kind\": \"ConfigMap\","
+							+ "\"metadata\": {\"name\": \"my-cm-create\",\"namespace\": \"test\"}}")))
+					.build();
+
+			// 2 - Define WorkFlowRequests
+			WorkFlowRequestDTO workFlowRequestGet = WorkFlowRequestDTO.builder().projectId(testProject.getId())
+					.workFlowName(workflowName).works(Arrays.asList(workGet)).build();
+			WorkFlowRequestDTO workFlowRequestCreate = WorkFlowRequestDTO.builder().projectId(testProject.getId())
+					.workFlowName(workflowName).works(Arrays.asList(workCreate)).build();
+			WorkFlowRequestDTO workFlowRequestUpdate = WorkFlowRequestDTO.builder().projectId(testProject.getId())
+					.workFlowName(workflowName).works(Arrays.asList(workUpdate)).build();
+
+			WorkFlowRequestDTO workFlowRequests[] = new WorkFlowRequestDTO[] { workFlowRequestGet,
+					workFlowRequestCreate, workFlowRequestUpdate };
+
+			// 3 - Execute WorkFlowRequests
+			for (WorkFlowRequestDTO workFlowRequest : workFlowRequests) {
+				WorkFlowResponseDTO execute = workflowApi.execute(workFlowRequest);
```

Comment:
pls note that this test will be broken one the patch for async execution is merged (which might be the problem of the latter to solve).

Reply:
there are many tests that will be broken. I hope you will implement it correctly and leave the default behavior as it is today - blocking/waiting

Reply:
It will be my responsibility to make sure the entire suite of tests is working properly. Changes will have to be introduced since the API will be changed, so the SDK is changed and therefore the tests will have to be updated.